src/PSRule.Rules.Azure/rules/Azure.Databricks.Rule.yaml

```
@@ -25,3 +25,43 @@ spec:
         isBoolean: true
       - field: properties.parameters.enableNoPublicIp.value
         equals: true
+
+---
+# Synopsis: Use non-Trial SKUs for Databricks workspaces.
+apiVersion: github.com/microsoft/PSRule/v1
+kind: Rule
+metadata:
+  name: Azure.Databricks.SKU
+  ref: AZR-000409
+  tags:
+    release: GA
+    ruleSet: 2024_01
+    Azure.WAF/pillar: Performance Efficiency
+spec:
+  type:
+  - Microsoft.Databricks/workspaces
+  condition:
+    allOf:
+      - field: sku.name
+        notEquals: trial
+
+---
+# Synopsis: Azure Databricks Workspaces should disable public network access.
+apiVersion: github.com/microsoft/PSRule/v1
+kind: Rule
+metadata:
+  name: Azure.Databricks.PublicAccess
+  ref: AZR-000410
+  tags:
+    release: GA
+    ruleSet: 2024_01
+    Azure.WAF/pillar: Security
+spec:
+  type:
+  - Microsoft.Databricks/workspaces
+  condition:
+    allOf:
+      - field: properties.publicNetworkAccess
+        exists: true
+      - field: properties.publicNetworkAccess
+        equals: Disabled
```

Don't have to change it. More just a call out that this is a simpler alternative:

```suggestion
  condition:
    field: properties.publicNetworkAccess
    equals: Disabled
```

Referencing a field with a specific value implies the property exists. PSRule first checks the property exists, then check that the value matches the expected value.